packages/website/components/page-templates/package-home.test.js

```diff
@@ -3,6 +3,9 @@ import titleCase from 'title-case';
 import PackageHomeWrapper, { Description } from './package-home';
 
 jest.mock('../navigation/package-nav-content', () => () => <div />);
+/* eslint-disable react/prop-types */
```

Comment:
Not blocking, just echoing:

`/* eslint-disable-next-line react/prop-types */` is slightly neater.